src/components/charts/SummaryPieChart/SummaryPieChart.css

```diff
@@ -49,4 +49,8 @@
 
 #recharts_measurement_span {
   display: none;
-}
\ No newline at end of file
+}
+
+.summary-pie-chart__toggle {
+  float: right;
+}
```

Comment:
also it would be good if we can add `font-size: 14px` in here, right now the font is too big so the text is overflowing to another line